frontend/src/components/InfoPanel.jsx

```diff
@@ -35,7 +35,7 @@ export default class InfoPanel extends React.Component {
       task.getCountryRestrictionDates(this.props.isoCode),
     ]);
 
-    if ((restrictionsDates.length != 0) & (this.props.isoCode != null)) {
+    if ((restrictionsDates != null) & (this.props.isoCode != null)) {
```

Comment:
We might also need to check that `restrictionsDates.first_restrictions_date` exists before trying to use it.